bukkit/src/main/java/net/william278/husksync/listener/BukkitEventListener.java

```diff
@@ -150,4 +161,14 @@ public void onPlayerTakeDamage(@NotNull EntityDamageEvent event) {
         }
     }
 
+    @EventHandler(priority = EventPriority.LOW, ignoreCancelled = true)
+    public void onPermissionCommand(PlayerCommandPreprocessEvent event) {
+        var commandArgs = event.getMessage().substring(1).split(" ");
```

Comment:
Use String; avoid using var in general (it's a neat Java 11 language feature and probably has its uses but not really a good idea for readability as a whole)

Reply:
Sure, no problem!